bigbluebutton-html5/imports/ui/components/polling/container.jsx

```diff
@@ -14,7 +14,7 @@ const propTypes = {
 const POLLING_ENABLED = Meteor.settings.public.poll.enabled;
 
 const PollingContainer = ({ pollExists, ...props }) => {
-  const currentUser = Users.findOne({ userId: Auth.userID });
+  const currentUser = Users.findOne({ userId: Auth.userID }, { fields: { presenter: 1 } });
```

Comment:
This call to Users should be in the Tracker and it could also just be using the poll service's `amIPresenter()` function.